modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/util/CustomCheckoutStep.java

```diff
@@ -157,26 +167,15 @@ public void render(
 			HttpServletResponse httpServletResponse)
 		throws Exception {
 
-		CommerceContext commerceContext =
-			(CommerceContext)httpServletRequest.getAttribute(
-				CommerceWebKeys.COMMERCE_CONTEXT);
+		PrintWriter printWriter = httpServletResponse.getWriter();
 
-		CommerceOrder commerceOrder = commerceContext.getCommerceOrder();
+		printWriter.write("<div>");
 
-		User currentUser = _userService.getCurrentUser();
+		String moduleName = "default from " + _renderURL;
```

Comment:
here is the render

Reply:
I suppose `_renderURL` contains the bare specifier and that's why it is failing. It should have an absolute URL I suppose. Something like `https://my-fancy-cdn.com/custom-checkout-step/index.js` or, if it was deployed from the Workspace it should be something like `/o/my-custom-checkout-step/index.js`...